app/views/projects/members/index.html.erb

```diff
@@ -85,15 +85,42 @@
                     <% end %>
                   </div>
                 </td>
-                <% if allowed_to?(:destroy_member?, @namespace) %>
+                <% if current_user == member.user || allowed_to?(:destroy_member?, @namespace) %>
                   <td class="px-4 py-3 text-right">
                     <%= viral_dropdown(icon: "ellipsis_vertical", classes: "member-settings-ellipsis", aria: { label: t(:'projects.members.index.actions.member_dropdown_aria_label', member: member.user.email) }) do |dropdown| %>
                       <%= dropdown.item(
-                        label: t(:"projects.members.index.remove"),
+                        label:
+                          (
+                            if current_user == member.user
+                              t(:"projects.members.index.leave_project")
+                            else
```

Comment:
**nitpick (blocking):** I think since you have multiple `current_user == member.user` you should create a partial for each.  This would require you to only need to have one clause.

Reply:
Definitely was a good idea..thanks! Updated in c77f10c5b6582c4430ebf914ac27cffbb6ce464b